RoboJackets/igvc-software
# Costmap Layers Refactor

# Description
This PR rearranges the costmap layers so that the sensor data is applied directly to the global costmap and the local costmap is a zoom in on the global costmap, instead of the local costmap "constructing" the global costmap

This PR does the following:
- changes the global costmap plugins to `traversability_layer`, `line_layer`, and `inflation_layer`
- changes the local costmap plugins to `rolling_layer`
- creates `rolling_layer`, which copies data from the global costmap that surrounds the robot

Fixes #655 

# Testing steps (If relevant)
## Test Case 1
1. Run a simulation
2. Open rviz
3. Set a goal

Expectation: The global costmap should update according to `traversability_layer` and `line_layer`. The local costmap should copy over the LETHAL cost cells.

# Self Checklist
